newIDE/app/src/ObjectEditor/Editors/SpriteEditor/DirectionTools.js

```diff
@@ -158,6 +163,9 @@ export default class DirectionTools extends Component<Props, State> {
               spritesContainer={direction}
               resourcesLoader={resourcesLoader}
               project={project}
+              onBlur={() => this.saveTimeBetweenFrames()}
```

Comment:
`onBlur` only works on text field. Flow is not warning you because Props of AnimationPreview are not exact, let me fix that.
Instead, move this to openPreview, to call `this.saveTimeBetweenFrames` when the argument is false (i.e: when the preview is closed, we save the time between frames).

Reply:
Yup, as expected, after using Flow exact typing: 
`Cannot create AnimationPreview element because property onBlur is missing in
Props [1] but exists in props [2].`

Reply:
@4ian ready now! :) I just realized its not really saving it. yes you're right- the onBlur is a built in parameter for the react module. When its a custom module we need to handle it manually, I sometimes take these things for granted too much